website/versioned_docs/version-25.x/JestObjectAPI.md

```diff
@@ -560,7 +560,7 @@ test('plays audio', () => {
 
 ### `jest.clearAllMocks()`
 
-Clears the `mock.calls`, `mock.instances` and `mock.results` properties of all mocks. Equivalent to calling [`.mockClear()`](MockFunctionAPI.md#mockfnmockclear) on every mocked function.
+Clears the `mock.calls`, `mock.instances`, `mock.contexts` and `mock.results` properties of all mocks. Equivalent to calling [`.mockClear()`](MockFunctionAPI.md#mockfnmockclear) on every mocked function.
```

Comment:
You should work with files from `/docs` directory. These are the ones for future releases. The docs under `/website` directory are for previous versions of Jest.

Reply:
Understood, I'll update the changes.

Reply:
Done